examples/src/main/java/com/vesoft/nebula/examples/database/TestPool.java

```diff
@@ -0,0 +1,56 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package com.vesoft.nebula.examples.database;
+
+import com.vesoft.nebula.client.graph.NebulaGraphConnection;
+import com.vesoft.nebula.client.graph.pool.NebulaGraphConnectionPool;
+import com.vesoft.nebula.client.graph.pool.NebulaGraphPoolConnectionFactory;
+import com.vesoft.nebula.client.graph.pool.config.NebulaConnectionPoolConfig;
+import com.vesoft.nebula.client.graph.pool.entity.LinkDomain;
+import java.util.Arrays;
+
+/**
+ * @author huangzhaolai-jk
+ * @version 1.0.0
+ * @Description TestPool is used for
+ * @Date 2020/3/24 - 15:00
+ */
+public class TestPool {
+
+    public static void main(String[] args) throws Exception {
+        test2();
+    }
+
+    public static void test1() throws Exception {
```

Comment:
where is using `test1`?

Reply:
It is just one of the test methods which is used in main method during the develop time

Reply:
maybe we can merge them into one